pkgs/games/openmw/default.nix

```diff
@@ -11,20 +12,24 @@ let
     };
   });
 in stdenv.mkDerivation rec {
-  version = "0.43.0";
+  version = "0.44.0";
   name = "openmw-${version}";
 
   src = fetchFromGitHub {
     owner = "OpenMW";
     repo = "openmw";
     rev = name;
-    sha256 = "1nybxwp77qswjayf0g9xayp4x1xxq799681rhjlggch127r07ifi";
+    sha256 = "0rxkw0bzag7qffifg28dyyga47aaaf5ziiccpv7p8yax1wglvymh";
   };
 
   enableParallelBuilding = true;
 
   nativeBuildInputs = [ pkgconfig ];
-  buildInputs = [ cmake boost ffmpeg qt4 bullet mygui openscenegraph_ SDL2 unshield openal libXt ];
+  buildInputs = [ cmake boost ffmpeg bullet mygui openscenegraph_ SDL2 unshield openal libXt ]
+  ++ [ (if useQt5 then qt5.qtbase else qt4) ];
```

Comment:
Any reason to not use qt5 unconditionally?

Reply:
We should just drop qt4 completely.

Reply:
It is still the default. I have found that is because a bug with the multi threading ( [https://github.com/OpenMW/openmw/pull/1525](https://github.com/OpenMW/openmw/pull/1525) ), and the upstream commit was rejected ( at 2017, two years ago, that may be outdated ). There is also this forum discussion about it ( [https://forum.openmw.org/viewtopic.php?f=6&t=5259](https://forum.openmw.org/viewtopic.php?f=6&t=5259) ), and it appear that the issue with multithreading isn't still corrected.

Reply:
Based on that thread, single threading isn't actually an issue and there seems to be consensus upstream to get off qt4 when possible. Furthermore, the distros I checked that carry openmw (debian, gentoo, opensuse, arch) are all building against qt5 and the closer we can be to the packaging on other distros, the easier troubleshooting will be.

Reply:
When doing a complete switch to qt5 is would be good to take https://nixos.org/nixpkgs/manual/#ssec-qt-applications into account

Reply:
I have finally force the use of Qt5 (as tes3mp use qt5), but made it in a separate commit, so it can be easily canceled.